### PR TITLE
Fixing Freeze Image problem

### PR DESCRIPTION
**Pull request by NimaSoroush**
- Fixing issue with freezing image
- Upgrading Chromy to version 0.4.7

### CODE REVIEW

File: src/chromyRunner.js [hunk 1]
```diff
@@ -105,7 +106,8 @@ const run = async (chromy, options, test) => {
       case actions.freezeImage:
         try {
           prefixedLogger.log(`Freezing image ${action.value} in browser`);
-          const result = await chromy.evaluate(() => freezeImage(action.value));
+          const strFunc = functionToString(freezeImage, { '{{selector}}': action.value });
+          const result = await chromy.evaluate(`return (${strFunc})()`);
```
Comment (danrr) [inline src/chromyRunner.js]: The `return (` and `)()` should be inserted by `functionToString`, you will always want to put those there, right?

File: src/freezeImage.test.js [hunk 2]
```diff
@@ -21,11 +21,22 @@ describe('Freeze Image', () => {
...
     freezeImage('test');
-    expect(document.querySelector).toHaveBeenCalledWith('test');
+    expect(document.querySelector).toHaveBeenCalledWith('{{selector}}');
+    expect(document.createElement).toHaveBeenCalledWith('canvas');
+    expect(getContext).toHaveBeenCalledWith('2d');
+    expect(drawImage).toHaveBeenCalledTimes(1);
+    expect(replaceChild).toHaveBeenCalledTimes(1);
+  });
+
+  it('calls document with correct value', () => {
```
Comment (danrr) [inline src/freezeImage.test.js]: is this a duplicate test?

File: src/freezeImage.js [hunk 3]
```diff
@@ -1,5 +1,5 @@
-const freezeImage = (selector) => {
-  const image = document.querySelector(selector);
+const freezeImage = () => {
+  const image = document.querySelector('{{selector}}');
```
Comment (danrr) [inline src/freezeImage.js]: I know i asked already but I forgot, why the `{{`?

File: src/freezeImage.test.js (hunk 2 above)
Comment (NimaSoroush) [inline src/freezeImage.test.js]: will fix it. was not suppose to be duplicated

File: src/helpers/functionToString.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
+import check from 'check-types';
+
+const escapeRegExp = str =>
+  str.replace(/[-[\]/{}()*+?.\\^$|]/g, '\\$&');
+
+const functionToString = (func, replaces = {}) => {
```
Comment (danrr) [inline src/helpers/functionToString.js]: Does this need `replaces` as a param?

**Pull request merged**